Kernel/init.cpp

```diff
@@ -180,11 +180,11 @@ extern "C" [[noreturn]] UNMAP_AFTER_INIT void init(BootInfo const& boot_info)
         (*ctor)();
     kmalloc_init();
     slab_alloc_init();
-    CommandLine::initialize();
     Memory::MemoryManager::initialize(0);
 
     load_kernel_symbol_table();
 
+    CommandLine::initialize();
```

Comment:
Maybe squash this into the "Don't use eternal heap for kernel symbol data" commit.

Reply:
Done.